variables.tf

```
@@ -20,6 +20,7 @@ variable "talos" {
       primary_dns_server   = string
       secondary_dns_server = string
       ntp_endpoint         = string
+      node_prefix = string
```

```suggestion
      node_prefix          = optional(string, "")
```
1. Should be optional as prefix shouldn't be a hard requirement
2. String should have empty default for people who don't want to use the prefix
3. Indentation